view/src/components/Profile.js

```diff
@@ -4,65 +4,40 @@ import { Link } from 'react-router-dom';
 import agent from '../agent';
 import ScheduleSelector from "react-schedule-selector";
 import { connect } from 'react-redux';
+import axios from 'axios';
+
 import {
   PROFILE_PAGE_LOADED,
   PROFILE_PAGE_UNLOADED
 } from '../constants/actionTypes';
 
-const EditProfileSettings = props => {
```

Comment:
I think we could have left the "edit profile" for now..

Reply:
So this is actually a ‘teacher’ profile, and edit profile isn’t necessary here. I’ll add another page for ‘user’ profile, where you can also edit.

Reply:
oh I see.. sounds good but can you rename it to TeacherProfile instead of Profile?

Reply:
Also, should this PR close https://github.com/MLH-Fellowship/zusammen/issues/21 ?